client.go

```diff
@@ -93,13 +94,16 @@ type requestInput struct {
 	Headers     map[string]string
 	QueryParams map[string]string
 	Body        interface{}
+	RawBody 	[]byte
```

Comment:
I don't believe we need the `RawBody` for the `requestInput` we don't use it when we use the api , we marshall the `Body` inside the function